test/helpers.js

```diff
@@ -0,0 +1,30 @@
+// Helper function to determine whether this browser is supported by 
+// impress.js or not. Copied from impress.js itself.
+var _impressSupported = function() {
```

Comment:
I suppose it was just because this is internal helper. If you want to suggest a more proper way of encapsulating this function into some helper namespace, I will be listening :-)